src/components/flat-table/flat-table-cell/flat-table-cell.component.js

```diff
@@ -1,26 +1,38 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { space } from '@styled-system/prop-types';
+
 import StyledFlatTableCell from './flat-table-cell.style';
 
 const FlatTableCell = ({
   align,
   children,
   colspan,
-  rowspan
+  rowspan,
+  py,
+  px,
+  ...rest
 }) => {
   return (
     <StyledFlatTableCell
       align={ align }
       data-element='flat-table-cell'
       colSpan={ colspan }
       rowSpan={ rowspan }
+      py={ py || '10px' }
```

Comment:
why do we use the default of `10px`? Should it be `8px` in line with the theme multiplier?

Reply:
FlatTableCell had a `padding: 10px 24px` rule before so I think I had to maintain it that way.

Reply:
Yes, fair enough. Though I think we should perhaps have another task to align it with the theme. I'll resolve all my comments relating to this below.